components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandler.java

```diff
@@ -116,6 +119,15 @@ public void handle(HttpServletRequest request, HttpServletResponse response, Aut
             return;
         }
         if (!graph.isBuildSuccessful()) {
+            if (LoggerUtils.isDiagnosticLogsEnabled()) {
+                Map<String, Object> params = new HashMap<>();
+                params.put(FrameworkConstants.RequestParams.ISSUER, context.getRelyingParty());
+                params.put(FrameworkConstants.RequestParams.TENANT_DOMAIN, context.getTenantDomain());
+                LoggerUtils.triggerDiagnosticLogEvent(
+                        FrameworkConstants.LogConstants.AUTHENTICATION_FRAMEWORK, params, LogConstants.FAILED,
+                        "Error while parsing the authentication script. Nested exception is: " + graph.getErrorReason(),
```

Comment:
Shall we also have an error code or similar reference, which we can mention in docs with further details and how to fix/recover

Reply:
This is tracked with https://github.com/wso2-enterprise/asgardeo-product/issues/12352